cmd/kraft/pkg/pkg.go

```diff
@@ -164,13 +182,46 @@ func (opts *Pkg) Run(cmd *cobra.Command, args []string) error {
 
 	var tree []*processtree.ProcessTreeItem
 
-	parallel := !config.G[config.KraftKit](ctx).NoParallel
+	// parallel := !config.G[config.KraftKit](ctx).NoParallel
 	norender := log.LoggerTypeFromString(config.G[config.KraftKit](ctx).Log.Type) != log.FANCY
 
+	exists, err := pm.Catalog(ctx,
+		packmanager.WithName(opts.Name),
+	)
+	baseopts := []packmanager.PackOption{}
+	if err == nil && len(exists) >= 1 {
+		if opts.strategy == packmanager.StrategyPrompt {
+			strategy, err := selection.Select[packmanager.MergeStrategy](
+				fmt.Sprintf("package '%s' already exists: how would you like to proceed?", opts.Name),
+				packmanager.MergeStrategies()...,
+			)
+			if err != nil {
+				return err
+			}
+
+			opts.strategy = *strategy
+		}
+
+		switch opts.strategy {
+		case packmanager.StrategyExit:
+			return fmt.Errorf("package already exists and merge strategy set to exit on conflict")
+
+		// Set the merge strategy as an option that is then passed to the
+		// package manager.
+		case packmanager.StrategyMerge, packmanager.StrategyOverwrite:
```

Comment:
Should this have a default case? Otherwise, should this be the default case?

Reply:
Will set to `default`.